frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -42,40 +44,53 @@ export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabP
   );
 };
 
-export const DefaultOverviewPage = connectToModel(
-  ({ kindObj: kindObject, item, customActions }) => {
-    if (!kindObject && !item?.obj) {
-      return null;
-    }
-    const resourceModel = kindObject || modelFor(referenceFor(item.obj));
-    return (
-      <div className="overview__sidebar-pane resource-overview">
-        <ResourceOverviewHeading
-          actions={[
-            ...(customActions ? customActions : []),
-            ...Kebab.getExtensionsActionsForKind(resourceModel),
-            ...common,
-          ]}
-          kindObj={resourceModel}
-          resources={item}
-        />
-        <div className="overview__sidebar-pane-body resource-overview__body">
-          <div className="resource-overview__summary">
-            <ResourceSummary resource={item.obj} />
-          </div>
+export const DefaultOverviewPage: React.FC<{ item: OverviewItem }> = ({ item }) => {
+  return (
+    <div className="overview__sidebar-pane resource-overview">
+      <div className="overview__sidebar-pane-body resource-overview__body">
+        <div className="resource-overview__summary">
+          <ResourceSummary resource={item.obj} />
         </div>
       </div>
-    );
-  },
-);
+    </div>
+  );
+};
+
+const DefaultSideBar: React.FC<{
+  kindObj: K8sKind;
+  item: OverviewItem;
+  customActions?: KebabAction[];
+}> = ({ kindObj, item, customActions }) => {
+  const { t } = useTranslation();
+
+  const tabs = [
+    {
+      name: t('public~Details'),
+      component: DefaultOverviewPage,
+    },
+  ];
+  const menuActions = [
+    ...(customActions ? customActions : []),
+    ...Kebab.getExtensionsActionsForKind(kindObj),
+    ...common,
+  ];
+  return (
+    <ResourceOverviewDetails
+      item={item}
+      kindObj={kindObj}
+      menuActions={customActions ? [...customActions, ...menuActions] : menuActions}
```

Comment:
Do you need to check for `customActions` again here? Its already getting added in `menuActions` above.